test/packages/good_v2/data_stream/routing_rules/routing_rules.yml

```diff
@@ -0,0 +1,35 @@
+routing_rules:
```

Comment:
Is this key redundant with the name of the file? Could we have directly a list?

Reply:
Yes, it looks like this key could be removed since this is now in its own file. I'll update the spec.